depthmapX/views/tableview/tableview.cpp

```diff
@@ -13,223 +14,206 @@
 // You should have received a copy of the GNU General Public License
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+#include "tableview.h"
 
-#include <QtCore/QEvent>
-#include <QtCore/QDebug>
+#include "mainwindow.h"
 
-#include <QtWidgets/QMenu>
-#include <QtWidgets/QLayout>
 #include <QKeyEvent>
-#include <QtWidgets/QMessageBox>
+#include <QtCore/QDebug>
+#include <QtCore/QEvent>
+#include <QtGui/QFocusEvent>
 #include <QtWidgets/QHeaderView>
+#include <QtWidgets/QLayout>
+#include <QtWidgets/QMenu>
+#include <QtWidgets/QMessageBox>
 #include <QtWidgets/QTableWidgetItem>
-#include <QtGui/QFocusEvent>
-
-#include "mainwindow.h"
-#include "tableview.h"
 
 #define ROW_HEIGHT 20
 #define PG_COUNT 40
 
 static bool in_update = 0;

```

Comment:
WTF?!
Why does this need to be a static? Should be a class member, right? 
Also, why a boolean used with integer semantics?